lte/gateway/c/session_manager/SessionState.h

```diff
@@ -84,6 +84,34 @@ class SessionState {
     std::vector<PolicyRule> gy_dynamic_rules;
     std::experimental::optional<AggregatedMaximumBitrate> ambr;
   };
+struct SessionCommonInfo {
```

Comment:
Is this supposed to be the enhanced version of `SessionInfo`?

Reply:
We can't  use old structre,  we can have a discussion on this after  2 days once the SMF-UPF message is converged. Yogesh,  raised we might changed the proto again.

Reply:
Migrated to older structure